sphinx/builders/linkcheck.py

```diff
@@ -75,18 +74,18 @@ def handle_starttag(self, tag, attrs):
                 self.found = True
 
 
-def check_anchor(f, anchor):
-    """Reads HTML data from a filelike object 'f' searching for *anchor*.
+def check_anchor(response, anchor):
+    """Reads HTML data from a response object 'response' searching for *anchor*.
```

Comment:
While we're here, make references to both arguments in the docstring uniform (`*response*`, or even better backticks.)